pymc3/sampling.py

```diff
@@ -1070,6 +1077,10 @@ def sample_posterior_predictive(trace, samples=None, model=None, vars=None, size
 
     model = modelcontext(model)
 
+    if varnames is not None:
+        if vars is not None:
+            raise Exception("Should not specify both vars and varnames arguments.")
+        vars = (model[x] for x in varnames)
     if vars is None:
```

Comment:
Add an `else` condition that says that `vars` is deprecated in favor of `varnames`

Reply:
Will do.

Reply:
This can become an else clause